spec/features/person_view_spec.rb

```diff
@@ -31,4 +31,38 @@
     expect(current_path).to eq(person_path(person))
     expect(page).to have_content('123-4567')
   end
+
+  it "has links to edit phone numbers" do
+    person.phone_numbers.each do |phone|
+      expect(page).to have_link('edit', href: edit_phone_number_path(phone))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.